mlflow/experiments.py

```diff
@@ -108,3 +108,18 @@ def rename_experiment(experiment_id, new_name):
     store = _get_store()
     store.rename_experiment(experiment_id, new_name)
     print("Experiment with id %s has been renamed to '%s'." % (experiment_id, new_name))
+
+
+@commands.command("csv")
+@EXPERIMENT_ID
+@click.option("--filename", type=click.STRING, required=True)
+def generate_csv_with_runs(experiment_id, filename):
+    # type: (str, str) -> None
+    """
+    Generate CSV with all runs for an experiment
+    """
+    runs = fluent.search_runs(
+        experiment_ids=experiment_id, index=False)
```

Comment:
I think you mean for the `index=False` to be in the `runs.to_csv()` call.

You can test linting errors using `./lint.sh` from the base of the `mlflow` directory

Reply:
Got it. I did the fix